apis/run/v1alpha3/clusterbootstrap_types.go

```diff
@@ -5,11 +5,15 @@ package v1alpha3
 
 import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	clusterapicondition "sigs.k8s.io/cluster-api/api/v1beta1"
 )
 
 // ClusterBootstrapStatus defines the observed state of ClusterBootstrap
 type ClusterBootstrapStatus struct {
 	ResolvedTKR string `json:"resolvedTKR,omitempty"`
+
+	Conditions clusterapicondition.Conditions `json:"conditions,omitempty"`
 }
```

Comment:
I think you have to implement getter & setter so the patch helper works correctly with conditions
Example from CAPI:
```go
// GetConditions returns the set of conditions for this object.
func (c *KubeadmConfig) GetConditions() clusterv1.Conditions {
	return c.Status.Conditions
}

// SetConditions sets the conditions on this object.
func (c *KubeadmConfig) SetConditions(conditions clusterv1.Conditions) {
	c.Status.Conditions = conditions
}
```

Reply:
Implemented though can't use ```Set``` due to the reason specified in this thead: https://github.com/vmware-tanzu/tanzu-framework/pull/1781#discussion_r834847931